src/main/java/blackjack/domain/card/Deck.java

```diff
@@ -0,0 +1,59 @@
+package blackjack.domain.card;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class Deck {
+	static final int INIT_SIZE = 52;
+
+	private final List<Card> cards;
+
+	public Deck(List<Card> cards) {
+		ArrayList<Card> copyDeck = new ArrayList<>(cards);
+		validate(copyDeck);
+		this.cards = copyDeck;
+	}
+
+	private void validate(List<Card> cards) {
+		validateDuplicate(cards);
+		validateSize(cards);
+	}
+
+	private void validateDuplicate(List<Card> cards) {
+		long distinctCount = cards.stream()
+			.distinct()
+			.count();
+
+		if (cards.size() != distinctCount) {
+			throw new IllegalArgumentException("카드는 중복될 수 없습니다.");
+		}
+	}
+
+	private void validateSize(List<Card> deck) {
+		if (deck.size() != INIT_SIZE) {
+			throw new IllegalArgumentException(String.format("덱의 사이즈가 %d장이 아닙니다.", INIT_SIZE));
+		}
+	}
+
+	public List<Card> drawCards(int count) {
+		List<Card> cards = new ArrayList<>();
+		for (int i = 0; i < count; i++) {
+			cards.add(drawCard());
+		}
+
+		return cards;
+	}
```

Comment:
```suggestion
	public List<Card> drawInitCards() {
		return List.of(drawCard(), drawCard());
	}
```
count는 항상 2일것 같은데 굳이 for문을 쓸 필요없을것 같습니다!

Reply:
블랙잭 게임 규칙은 초기 2장 카드를 던져주는 것이 맞지만 확장성을 살~짝 고려해서 저렇게 구현해보았는데요.

오히려 초기 카드가 2장이라는 사실을 알아야 코드를 이해할 수 있도록 만들어져서 도메인 지식이 적은 사람들에게는 불편함을 줄 수도 있는 코드가 되어버린 것 같네요..
체킹 감사드립니다!